pkg/route/api/v1/conversion.go

```diff
@@ -13,6 +13,9 @@ func init() {
 			obj.To.Kind = "Service"
 		},
 		func(obj *TLSConfig) {
+			if len(obj.Termination) == 0 {
+				obj.Termination = TLSTerminationEdge
```

Comment:
> It doesn't matter what we want here - if we shipped an API that
> accepted an empty value and behaved one way, then realized it need to
> be set for clarity, a default value is exactly how we solve that.

Spoke with @smarterclayton in person.  Here's the net:
 1. We can add validation to reject `TLS.Termination==""`
 1. If the `TLSConfig` does not have a `Termination` and does not have a `DestinationCACertificate` and `validateInsecureEdgeTerminationPolicy` would return ok, then default `TLS.Termination` to edge.  This is basically, "if the `TLSConfig` has a shot at working for edge termination.

@smarterclayton one thing I forgot to ask.  If the values are present, but broken, will the unencrypted route still work?